### ISSUE TITLE
As a timetable planner, I want special days to be indicated consistently in the day type view and versions

### ISSUE THREAD

**Issue by juhakolari** [HSLdevcom/jore4#1623]
As on the versions page, the Day type view for a special day should indicate which reference day is used “Maanantai”/”Ajetaan kuten sunnuntai”.

Versions:

![Image](https://github.com/HSLdevcom/jore4/assets/77674841/abd519b3-6b74-42d0-91f0-17f761c6b9a5)

Day type view (missing the text):

![Image](https://github.com/HSLdevcom/jore4/assets/77674841/c347c681-8926-47ed-b194-d113e11810a4)
